src/resources/styles.ts

```diff
@@ -94,6 +94,72 @@ export const haStyle = css`
   .card-actions .warning {
     --mdc-theme-primary: var(--google-red-500);
   }
+
+  .layout.horizontal,
```

Comment:
Do we really want to copy these all over instead of just using flexbox styles? Things like `-ms-` are not needed because Edge is based on Chrome nowadays.